Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,510 @@
+to Simulink. There will also be an arduino to control the use of the solenoid. The solenoid will be activated from the Simulink program, which will coincide with
+the beginning of data acquisition.
+
+The system will also be displayed at Imagine RIT 2018. This will include a full poster and data representation.
+
+\subsubsection{Non-physical Deliverables}
+\label{subsubsec: non-physical deliverables}
+
+The non-physical deliverables of this project will be the project culmination report which will include a very detailed examination of the projects
+results. These results include the expected performance versus the test performance, the nozzle dimension polynomial approximation, the structure performance, and the
+propulsion system performance. The structure performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This specifically concerns the
+specific impulse. It will also concern the expansion of the flow, as the maximum thrust will occur when the exit pressure equals the ambient pressure and thus the shocks
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing. This was performed in the Spring 2017
+semester for flow visualization. The procedure was; turn the lights off, put the nozzle on the air hose, put a white piece of paper on a table, put it parallel to the table, initiate the cold burn, use a phone light
+to project the flow onto the white paper below it. Then distance between the nozzle and the light and the table were toggled until the best focus point was found. That is how the team
+was able to get the picture below.
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/nozzle-shocks[2].png}
+  \caption{The flow is seen here exiting a small 3D printed supersonic nozzle. The procedure was not unlike schlieren lensing, but the clarity was much lower.}
+  \label{fig: nozzle shocks}
+\end{figure}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+
+There is a preliminary plan for the semester that is displayed in further detail in Appendix A. The following list will display the description and timing
```

Comment:
`in further detail in Appendix A.`
Change this to a cross reference. Hard coding appendix numbers/letters, figure numbers, table numbers, or citations is not correct. If your element label changes in the future, you'd have to change everything manually--that's not what we want.

I think `\autoref` will do what you want. This command comes with the `hyperref` package.

`The following list will display the description and timing`
Omit this sentence.